src/presentation/ui/page/databases.templ

```diff
@@ -0,0 +1,299 @@
+				htmx.ajax(
+					'POST',
+					'/api/v1/services/installables/',
+					{
+						swap: 'none',
+						values: { name: databaseType },
+					},
+				);
+			},
+			get shouldDisableCreateDatabaseSubmitButton() {
+				return this.database.name == ''
+			},
+			get shouldDisableCreateDatabaseUserSubmitButton() {
+				return this.database.name == '' || this.databaseUser.username == '' || this.databaseUser.password == ''
+			},
+
+			// Modal states
+			isCreateDatabaseModalOpen: false,
+			openCreateDatabaseModal() {
+				this.resetPrimaryStates();
+
+				this.isCreateDatabaseModalOpen = true;
+			},
+			closeCreateDatabaseModal() {
+				this.isCreateDatabaseModalOpen = false;
+			},
+			isDeleteDatabaseModalOpen: false,
+			openDeleteDatabaseModal(databaseName) {
+				this.resetPrimaryStates();
+
+				this.database.name = databaseName;
+				this.isDeleteDatabaseModalOpen = true;
+			},
+			closeDeleteDatabaseModal() {
+				this.isDeleteDatabaseModalOpen = false;
+			},
+			deleteDatabaseElement(databaseType) {
+				htmx.ajax(
+					'DELETE',
+					'/api/v1/database/' + databaseType + '/' + this.database.name + '/',
+					{swap: 'none'},
+				);
+				this.closeDeleteDatabaseModal();
+			},
+			isCreateDatabaseUserModalOpen: false,
+			openCreateDatabaseUserModal() {
+				this.resetPrimaryStates();
+
+				this.isCreateDatabaseUserModalOpen = true;
+			},
+			closeCreateDatabaseUserModal() {
+				this.isCreateDatabaseUserModalOpen = false;
+			},
+			createDatabaseUser(databaseType) {
+				htmx.ajax(
+					'POST',
+					'/api/v1/database/' + databaseType + '/' + this.database.name + '/user/',
+					{
+						swap: 'none',
+						values: {
+							username: this.databaseUser.username,
+							password: this.databaseUser.password,
+							privileges: this.databaseUser.privileges,
+						}
+					},
+				);
+				this.closeCreateDatabaseUserModal();
+			},
+			isDeleteDatabaseUserModalOpen: false,
+			openDeleteDatabaseUserModal(databaseName, databaseUsername) {
+				this.resetPrimaryStates();
+
+				this.database.name = databaseName;
+				this.databaseUser.username = databaseUsername;
+				this.isDeleteDatabaseUserModalOpen = true;
+			},
+			closeDeleteDatabaseUserModal() {
+				this.isDeleteDatabaseUserModalOpen = false;
+			},
+			deleteDatabaseUserElement(databaseType) {
+				htmx.ajax(
+					'DELETE',
+					'/api/v1/database/' + databaseType + '/' + this.database.name + '/user/' + this.databaseUser.username + '/',
+					{swap: 'none'},
+				);
+				this.closeDeleteDatabaseUserModal();
+			},
+		}))
+	})
+}
+
+templ DatabasesIndex(selectedDbTypeSummary presenterDto.SelectedDatabaseTypeSummary) {
+	@DatabasesIndexLocalState()
+	<div class="flex flex-col" x-data="databases">
+		<div class="mb-6 flex flex-row items-center justify-between">
+			@componentStructural.PageTitle(
+				"Databases", "Manage your database services by adding and deleting databases or users.", "ph-database",
+			)
+		</div>
+		<div
+			id="databases-content"
```

Comment:
databases-page-content, no? content alone is quite generic here

Reply:
Done!